omero_figure/scripts/omero/figure_scripts/Figure_To_Pdf.py

```diff
@@ -60,6 +60,11 @@
     logger.error("Reportlab not installed. See"
```

Comment:
The message should be reviewed since we no longer recommend to install from pypi